src/Swoole/SwooleClient.php

```diff
@@ -230,7 +230,11 @@ protected function sendResponseContent(OctaneResponse $octaneResponse, SwooleRes
 
         $content = $octaneResponse->response->getContent();
 
-        $length = strlen($content);
+        if ($length = strlen($content) === 0) {
```

Comment:
```suggestion
        if (($length = strlen($content)) === 0) {
```

without wrapping parenthesis, `$length` would become a boolean

Reply:
I would just leave it a 2-liner :P